support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSignUp.jsx

```diff
@@ -1,228 +1,50 @@
-import { TextInput } from '@guardian/src-text-input';
-import { SvgArrowRightStraight } from '@guardian/src-icons';
+import React, { useEffect } from 'react';
 import ActionContainer from './components/ActionContainer';
 import ActionHeader from './components/ActionHeader';
 import ActionBody from './components/ActionBody';
-import ExpandableContainer from './components/ExpandableContainer';
 import BulletPointedList from './components/BulletPointedList';
 import SvgPersonWithTick from './components/SvgPersonWithTick';
-import { checkEmail } from 'helpers/forms/formValidation';
-import {
-  setPasswordError as setPasswordErrorAction,
-  updatePassword as updatePasswordAction,
-  type Action,
-} from '../../contributionsLandingActions';
-import { setPasswordGuest } from 'helpers/forms/paymentIntegrations/readerRevenueApis';
-import styles from './styles';
 import {
   OPHAN_COMPONENT_ID_SIGN_UP,
-  OPHAN_COMPONENT_ID_READ_MORE_SIGN_UP,
 } from './utils/ophan';
-import { trackComponentClick, trackComponentLoad } from 'helpers/tracking/behaviour';
-
-const bodyText = css`
-  ${body.small()};
-`;
+import { trackComponentLoad } from 'helpers/tracking/behaviour';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
 
-const expandableContainer = css`
+const listContainer = css`
   margin-top: ${space[4]}px;
 `;
 
-const form = css`
-  margin-top: ${space[5]}px;
-
-  & > * + * {
-    margin-top: ${space[5]}px;
-  }
-`;
-
-const buttonContainer = css`
-  margin-top: ${space[6]}px;
-`;
-
-const mapStateToProps = state => ({
-  password: state.page.form.setPasswordData.password,
-  guestAccountCreationToken: state.page.form.guestAccountCreationToken,
-  passwordError: state.page.form.setPasswordData.passwordError,
-});
-
-function mapDispatchToProps(dispatch: Dispatch<Action>) {
-  return {
-    updatePassword: (event: Event) => {
-      if (event.target instanceof HTMLInputElement) {
-        dispatch(updatePasswordAction(event.target.value));
-      }
-    },
-    setPasswordError: (passwordError: boolean) => {
-      dispatch(setPasswordErrorAction(passwordError));
-    },
-  };
-}
-
-const checkPassword = (password: string) =>
-  password.length >= 6 && password.length <= 72;
-
-const PASSWORD_ERROR_MESSAGE =
-  'Please enter a password between 6 and 72 characters long';
-
-type ContributionThankYouSignUpProps = {|
-  csrf: Csrf,
-  email: string,
-  password: string,
-  guestAccountCreationToken: string,
-  updatePassword: Event => void,
-  passwordError: boolean,
-  setPasswordError: boolean => void
-|};
-
-const ContributionThankYouSignUp = ({
-  csrf,
-  email,
-  password,
-  guestAccountCreationToken,
-  updatePassword,
-  setPasswordError,
-  passwordError,
-}: ContributionThankYouSignUpProps) => {
-  const [isExpanded, setIsExpanded] = useState(false);
-  const [hasBeenCompleted, setHasBeenInteractedWith] = useState(false);
-
+const ContributionThankYouSignUp = () => {
```

Comment:
This component used to ask the user to enter a password. Now it just tells them to check their email